src/interchange/src/json.rs

```diff
@@ -77,7 +77,7 @@ impl fmt::Debug for JsonEncoder {
                 "schema",
                 &format!(
                     "{:?}",
-                    build_row_schema_json(&self.value_columns, "schema", &BTreeMap::new())
+                    build_row_schema_json(&self.value_columns, "schema", &BTreeMap::new(), false)
```

Comment:
ditto here––using boolean params in the outermost API isn't ideal

Reply:
I am leaving this one in. In the following avro updates, there will be more configs like user defined comments and eventually user defined default values which should live in a struct along with this flag.